magento/pwa-studio
# PWA-989: Real Data Wishlists and Wishlist Create

## Description
- Hook up the Wishlist page to pull actual real data from GraphQL (remove all the mock data)
- Hook up the Create Wishlist button on the Favorites / Wishlist page so that it actually fires the `createWishlist` GQL mutation (and updates the page)

## Related Issue





Closes PWA-989

## Acceptance
- Wishlist page should use real data from backend
- Should be possible to create new wishlist, when new wishlist created - it appears on the page



### Verification Stakeholders
@dpatil-magento
### Specification



### Verification Steps


 1. Create a Private wish list and check the UI.
 2. Create a Public wish list and check the UI.
 3. Verify existing list are shown when user log-in.
 4. Verify Private and Public lists with Max and Min characters and check the UI.
 5. Verify functionality of Cancel Button
 6. Verify wishlists are not accessible for guest users.
 7. Verify Auth user can access wish list of their account only.
 8. Verify UI on other browsers and devices (only if any UI changes)
 9. Run MFTF

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
